ignite/pom.xml

```diff
@@ -0,0 +1,264 @@
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.lucene</groupId>
+					<artifactId>lucene-core</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+<!-- GridGain 7.5.x -->
+<!--  
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-spring</artifactId>
+            <version>1.5.1-b1</version>
+            <exclusions>
+            	<exclusion>
+            		<groupId>commons-logging</groupId>
+            		<artifactId>commons-logging</artifactId>	
+            	</exclusion>
+            </exclusions>
+		</dependency>    
+		<dependency>
+			<groupId>commons-logging</groupId>
+			<artifactId>commons-logging</artifactId>
+			<version>1.1.3</version>
+		</dependency>    
+		<dependency>
+			<groupId>org.apache.ignite</groupId>
+			<artifactId>ignite-jta</artifactId>
+			<version>1.5.1-b1</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.ignite</groupId>
+			<artifactId>ignite-indexing</artifactId>
+			<version>1.5.1-b1</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.lucene</groupId>
+					<artifactId>lucene-core</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+-->
+    </dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<configuration>
+					<forkMode>once</forkMode>
+					<skipExec>${skipUnitTests}</skipExec>
+					<enableAssertions>false</enableAssertions>
+					<dependenciesToScan>
+						<dependency>org.hibernate.ogm:hibernate-ogm-core</dependency>
+					</dependenciesToScan>
+<!--  					
+					<includes>
+						<include>org/hibernate/ogm/backendtck/id/AutoIdGeneratorTest.java</include>
+						<include>org/hibernate/ogm/backendtck/id/AutoIdGeneratorWithSessionTest.java</include>
+						<include>org/hibernate/ogm/backendtck/id/DuplicateIdDetectionTest.java</include>
+						<include>org/hibernate/ogm/backendtck/id/IdentityIdGeneratorTest.java</include>
+						<include>org/hibernate/ogm/backendtck/id/SequenceIdGeneratorTest.java</include>
+					</includes>
+-->					
+					<excludes>
+						<exclude>org/hibernate/ogm/backendtck/massindex/*</exclude>
+						<exclude>org/hibernate/ogm/backendtck/optimisticlocking/*</exclude>
+						<exclude>org/hibernate/ogm/backendtck/perf/*</exclude>
+<!--
+						<exclude>org/hibernate/ogm/backendtck/queries/QueriesWithEmbeddedCollectionTest.java</exclude>
+						<exclude>org/hibernate/ogm/backendtck/queries/CompositeIdQueriesTest.java</exclude>
+						<exclude>org/hibernate/ogm/backendtck/queries/QueriesWithEmbeddedTest.java</exclude>
+						<exclude>org/hibernate/ogm/backendtck/queries/QueriesWithEmbeddedTest.java</exclude>
+						<exclude>org/hibernate/ogm/backendtck/queries/SimpleQueriesTest.java</exclude>
+						<exclude>org/hibernate/ogm/backendtck/queries/JpaQueriesTest.java</exclude> 
+-->
+					</excludes>
+					<argLine> -Dtest=IgniteBackendTckHelper</argLine>
```

Comment:
Do you need to pass the test variable and use the exclude, couldn't you just avoid using the excludes clause?